requirements/detection_test.txt

```diff
@@ -1 +1,2 @@
+torchvision>=0.13

```

Comment:
```suggestion
```
it is already in detection

Reply:
I think this should stay to make sure all tests are ran

Reply:
they run in the latest configuration